worker/pkg/benthos/sql/output_sql_insert.go

```diff
@@ -357,31 +282,28 @@ func (s *pooledInsertOutput) WriteBatch(ctx context.Context, batch service.Messa
 			return err
 		}
 
-		err = s.RetryInsertRowByRow(ctx, builder, insertQuery, s.columns, rows, columnDefaults)
+		err = s.RetryInsertRowByRow(ctx, builder, rows)
 		if err != nil {
 			return err
 		}
+		return err
```

Comment:
Hm, is this intending to return the outer error? if so, the err that is used for RetryInsertRowByRow should be renamed.